share/spice/public_holidays/record.handlebars

```diff
@@ -3,8 +3,8 @@
         {{#each holidays}}
             <tr class="record__row  {{#if ../meta.options.rowHighlight}}record__row--highlight{{/if}}">
```

Comment:
We can remove the `{{if}}...` here -- that's part of the template and we don't need it here